container/build_tar.py

```diff
@@ -293,29 +310,31 @@ def main(unused_argv):
     for file_owner in FLAGS.owners:
       (f, owner) = file_owner.split('=', 1)
       (user, group) = owner.split('.', 1)
-      if f[0] == '/':
-        f = f[1:]
+      f.lstrip('/')
       ids_map[f] = (int(user), int(group))
 
   # Add objects to the tar file
   with TarFile(FLAGS.output, FLAGS.directory, FLAGS.compression) as output:
-    for f in FLAGS.file:
-      (inf, tof) = f.split('=', 1)
+    def file_attributes(filename):
       mode = default_mode
       ids = default_ids
       names = default_ownername
-      map_filename = tof
-      if tof[0] == '/':
-        map_filename = tof[1:]
-      if map_filename in mode_map:
-        mode = mode_map[map_filename]
-      if map_filename in ids_map:
-        ids = ids_map[map_filename]
-      if map_filename in names_map:
-        names = names_map[map_filename]
-      output.add_file(inf, tof, mode, ids, names)
+      filename.lstrip('/')
+      if filename in mode_map:
+        mode = mode_map[filename]
```

Comment:
`mode_map.get(filename, default_mode)` is cleaner.

Reply:
done